.github/workflows/ScreenShotTest.yml

```diff
@@ -0,0 +1,54 @@
+on:
+  pull_request:
+
+jobs:
+  screenshot-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
```

Comment:
[nit] Would you keep using v1 action? or just forget changing to v3?
I can see the `v3.x.x` in the [action releases](https://github.com/actions/setup-java/releases)

Reply:
I'd like to know the your thought if you have. 🙇

Reply:
Yes, you are right, it should be v3.x.x.:pray: